CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+# Contributing 
+
+We welcome feedback and help with Psammead. By participating in Psammead, you agree to abide by the [Code of Conduct](https://github.com/BBC-News/psammead/blob/latest/CODE_OF_CONDUCT.md). Please take a moment to read it.
+
+We appreciate all forms of contribution - not just code. Contribution can include adding documentation, requesting clarification, making typo corrections and much more.
+
+Psammead has a "Definition of Done" which applies to changes proposed to its main branch, `latest`. When you make a pull request, please check that your contribution matches this.
+
+## Definition of Done
+
+**Component Structure**
+
+A component should have this file structure:
+
+```
+├── README.md
+├── CHANGELOG.md
+├── __snapshots__
+│   └── index.test.jsx.snap
+├── index.jsx
+├── index.stories.jsx
+├── index.test.jsx
+├── package.json
+└── package-lock.json
+```
+
+The `__snapshots__` directory is optional.
+
+
+**package.json**
+
+The `package.json` is namespaced with `@bbc/psammead-*` e.g. `@bbc/psammead-headings`.
+
+Include the following authorship and repository information in your package:
+
+```
+  "author": {
+    "name": "BBC Psammead Maintainers",
```

Comment:
What if another bbc team makes their own? we wouldn't be the author right? Probably one for slack

Reply:
Makes their own package. If bbc sport make a football table, we wouldnt be the author

Reply:
I guess just add a note above this saying "replace team name and email address if appropriate".